app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -735,6 +743,73 @@ public void displayHideCategorySearch() {
         }
     }
 
+    @OnClick(R.id.coordinate_edit)
+    public void onUpdateCoordinatesClicked(){
+        goToLocationPickerActivity();
+    }
+
+    /**
+     * Start location picker activity with a request code and get the coordinates from the activity.
+     */
+    private void goToLocationPickerActivity() {
+        double latitude = 37.773972;
+        double longitude = -122.431297;
```

Comment:
Are these necessary? If yes, please add a comment explaining why.

Reply:
Yes, those are default coordinates. If any file doesn't contain any location. By opening the location picker user will be first by default point on these coordinates. Then users can adjust their preferences at the same time.
You can suggest any default location. If you have any specific preference.